experiments/veo-app/common/utils.py

```diff
@@ -0,0 +1,10 @@
+def print_keys(obj, prefix=""):
+    """Recursively prints keys of a JSON object."""
```

Comment:
I think we should add a base case to error out the recursion when the obj is None, so it doesn't get stuck in an infinite loop.

Reply:
Looks great.